src/utils/mk64.py

```diff
@@ -17,7 +20,8 @@ def compare_records(original, newrecords):
                 print('New top 5 race time!')
                 break
 
-        if old_times['track-record'].records[5] != new_times['track-record'].records[5]:
+        best_lap_record = new_times['track-record'].records[5]
+        if old_times['track-record'].records[5] != best_lap_record and best_lap_record != None:
             # Commented these out because not being used atm
             # track_name = old_times['name']
             # racer_name = racer_ids[nr_re['character']]
```

Comment:
The line below, should use the new variable best_lap_record

Reply:
Done.